### ISSUE TITLE
how to tune hyperparameters using optuna or raytune

### ISSUE THREAD

**Issue by nikshrimali** [UKPLab/sentence-transformers#2247]
I want to finetune the MiniLM model and tune the hyperparameters of the same, but the model.fit function doesn't return any loss. Nor does it shows any performance metrics while training the model. What do you suggest in this case?
